errors.yaml

```
@@ -48,12 +52,24 @@ errors:
     description: "The Content-Type header does not match the request body payload format or the format is invalid."
   - code: missing_payload
     description: "The Content-Type header was specified, but no request body was sent to the server or the request body is empty."  
-  - code: dump_already_in_progress
-    description: "A dump creation is already in progress and a new one can't be triggered until the previous dump creation is not finished."
+  - code: dump_already_processing
+    description: "Dump creation is already in progress. A new dump creation process can be triggered when the current one is complete."
```

Minor rewording suggestion, don't feel strongly about it:

```suggestion
description: "Dump creation is already in progress. A new dump creation process can be triggered after the current one has been completed."
```